UnityProject/Assets/Scripts/Objects/Doors/Modules/CrowbarModule.cs

```diff
@@ -11,177 +11,162 @@ public class CrowbarModule : DoorModuleBase
 		[SerializeField] [Tooltip("Base time it takes to pry this door.")]
 		private float pryTime = 4.5f; //TODO calculate time with a multiplier from the tool itself
 
-		[SerializeField] private AddressableAudioSource prySound = null;
-
 		[SerializeField] [Tooltip("Can you crowbar pry the door when there no power")]
 		private bool crowbarRequiresNoPower = true;
 
-		private string soundGuid = "";
-
 		private string doorName;
+		private WeldModule weldModule;
 
 		protected override void Awake()
 		{
 			base.Awake();
 
 			doorName = transform.parent.gameObject.ExpensiveName();
+
+			weldModule = GetComponentInChildren<WeldModule>();
 		}
 
 		public override void OpenInteraction(HandApply interaction, HashSet<DoorProcessingStates> States)
 		{
-			if (interaction is not {Intent: Intent.Help}) return;
-			//If the door is powered, only allow things that are made to pry doors. If it isn't powered, we let crowbars work.
-
-			if (Validations.HasItemTrait(interaction.HandObject, CommonTraits.Instance.CanPryDoor) ||
-			    Validations.HasItemTrait(interaction.HandObject, CommonTraits.Instance.Crowbar))
+			//Require help intent to pry doors
+			if (interaction is { Intent: Intent.Help })
 			{
-				if ((crowbarRequiresNoPower && master.HasPower) &&
-				    (Validations.HasItemTrait(interaction.HandObject, CommonTraits.Instance.CanPryDoor) == false))
+				//If its hands that can pry doors, attempt to open the door
+				//TODO Currently hard coded to not allow larva, when prying with hands is moved to body parts just have larva not have the pry ability and update this
+				if (interaction.HandObject == null
+					&& interaction.PerformerPlayerScript.PlayerTypeSettings.CanPryDoorsWithHands &&
+					(interaction.PerformerPlayerScript.TryGetComponent<AlienPlayer>(out var alienPlayer) == false ||
+					 alienPlayer.IsLarva == false))
```

Comment:
Not something for this PR as aliens are there whole own bag of worms. But noting for future this shouldn't be hardcoded for Aliens/Alien Larvae. Would be better for aliens to just alter the CanPryDoorsWithHands directly on a stage by stage basis

Reply:
Yeah, this is how it was already implemented in the original CrowbarModule code with a "TODO: Fix this later".  I am not familiar enough with the current state of the Xenos code so I decided to let it be for now.